packages/gui/package.json

```
@@ -150,6 +150,7 @@
     "bech32": "2.0.0",
     "bignumber.js": "9.0.2",
     "byte-size": "8.1.0",
+    "chokidar": "^3.5.3",
```

Let's pin the version like we do with the others

```suggestion
    "chokidar": "3.5.3",
```